### PR TITLE
Standalone helm support for PowerFlex

### PR DESCRIPTION
**Pull request by KshitijaKakde**
#### Is this a new chart?

Yes/No

#### What this PR does / why we need it:
This PR address , standalone helm support for PowerFlex , it updates all the required files in csi-vxflexos 
#### Which issue(s) is this PR associated with:

- #Issue_Number
- https://github.com/dell/csm/issues/877

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by meggm — COMMENTED** [review]
Please include the GitHub issue link in the PR description.

**Comment by KshitijaKakde** [comment]
> Please include the GitHub issue link in the PR description.

Done

File: charts/csi-vxflexos/Chart.yaml
Comment (VamsiSiddu-7) [inline charts/csi-vxflexos/Chart.yaml:1]: @KshitijaKakde you can keep this file as it is. Looking at the changes i see the placement of the filelds is different here. There were no changes to the values.

File: charts/csi-vxflexos/Chart.yaml [hunk 1]
```diff
@@ -16,5 +16,4 @@ keywords:
 maintainers:
 - name: DellEMC
 sources:
-- https://github.com/dell/csi-vxflexos
```
Comment (VamsiSiddu-7) [inline charts/csi-vxflexos/Chart.yaml:19]: add the removed new line back

Comment (KshitijaKakde) [reply charts/csi-vxflexos/Chart.yaml:19]: Done

File: charts/csi-vxflexos/values.yaml [hunk 2]
```diff
@@ -245,6 +279,21 @@ node:
     # Default value: false
     enabled: false
 
+# Storage Capacity Tracking
+# Note: Capacity tracking is supported in kubernetes v1.24 and above, this feature will be automatically disabled in older versions.
+storageCapacity:
```
Comment (niranjan-n1) [inline charts/csi-vxflexos/values.yaml:284]: Add this in container-storage-modules chart as well

**Pull request closed without merging**